frontend/ui/network/manager.lua

```diff
@@ -243,6 +245,93 @@ function NetworkMgr:getBeforeWifiActionMenuTable()
        }
    }
 end
+function NetworkMgr:getSSHMenuTable()
```

Comment:
I'm thinking this would be more appropriate as a plugin than as part of the core.

Reply:
Sure, make sense, will look into it.